apiserver/facades/agent/uniter/secrets.go

```diff
@@ -78,8 +75,10 @@ func (u *UniterAPI) createSecret(ctx context.Context, arg params.CreateSecretArg
 
 	params := secretservice.CreateCharmSecretParams{
 		Version: secrets.Version,
-		// Secret accessor not needed when creating a secret since we are using the secret owner.
-		UpdateCharmSecretParams: fromUpsertParams(arg.UpsertSecretArg, secretservice.SecretAccessor{}, token),
+		UpdateCharmSecretParams: fromUpsertParams(arg.UpsertSecretArg, secretservice.SecretAccessor{
```

Comment:
The accessor is now required for checking inside the service. This knowledge was previously encapsulated in the token.